models/challenges.js

```diff
@@ -35,6 +36,33 @@ const fetchChallenges = async () => {
   }
 }
 
+/**
+ * Fetch the <user object> from participants array
+ * @param {Array} participants
+ * @returns {Promise<challengesModel|Array>}
+ */
+const fetchParticipants = async (participants) => {
+  try {
+    const fetchedparticipants = []
+    while (participants.length) {
+      const currentParticipants = participants.splice(0, 10)
+      const getParticipants = await userModel.where(admin.firestore.FieldPath.documentId(), 'in', currentParticipants).get()
```

Comment:
what's `admin.firestore.FieldPath.documentId()` doing?

Reply:
as .doc() directly gives the document, but I wanted to search for document which resembles from any of the id from array, I used this.